### ISSUE TITLE
咨询关于mipnerf的一些问题

### ISSUE THREAD

**Issue by AlphaPlusTT** [hjxwhy/Block-NeRF#2]
你好我在你的mipnerf-pl上提了一个问题就是我前段时间在我之前写的那个mipnerf-pytorch上又尝试了一些复现和修改，但是性能跟jax版本还是差了很多，我看您的mipnerf-pl复现的很成功，请问有没有一些建议能让我参考一下？

**Comment by hjxwhy** [comment]
我仔细看了一下你写的，你这里写错了https://github.com/AlphaPlusTT/mipnerf-pytorch/blob/c87326752b69b872b432484add28eacfb4e7a8e2/models/mip.py#L326，应该是这样
`trans = torch.exp(-torch.cat([
        torch.zeros_like(density_delta[..., :1]),
        torch.cumsum(density_delta[..., :-1], dim=-1)
    ], dim=-1))`
你少了一个负号

**Comment by AlphaPlusTT** [comment]
多谢，真是醉了
